frontend/pendingTranslations.ftl

```diff
@@ -394,6 +394,21 @@ faq-question-bulk-trackerremoval-answer = You can only turn tracker removal on a
 faq-question-trackerremoval-breakage-question = Why do my emails look broken?
 faq-question-trackerremoval-breakage-answer-2 = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images and links. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
 
+## VPN and Relay Bundle Banner
+#   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
+#   $old_price (string) - the outdated monthly cost (including currency symbol) for a given plan. This value has a strikethrough.
+#   #savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+bundle-banner-header = { -brand-name-firefox-relay } with <vpn-logo>{ -brand-name-mozilla-vpn }</vpn-logo>
+bundle-banner-subheader = Security, reliability and speed — on every device, anywhere you go.
+bundle-banner-body = Surf, stream, game, and get work done while maintaining your privacy online. Whether you’re traveling, using public Wi-Fi, or simply looking for more online security, we will always put your privacy first.
+bundle-banner-1-year-plan = 1 year plan: <b>{ -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn }</b>
+bundle-price-monthly = Monthly: <monthly-price>{ $monthly_price }</monthly-price>
+bundle-price-save-amount = Save { $savings } <outdated-price>Normally { $old_price }</outdated-price>
+bundle-banner-cta = Get { -brand-name-mozilla-vpn } and { -brand-name-relay }
+bundle-feature-one = More than 400 servers
+bundle-feature-two = More than 30 countries
+bundle-feature-three = Fast and secure network
```

Comment:
Really cool that you managed to make these localisable btw! And they're even quite resilient to longer sentences, nice job :clap: